docs/api-docs/customers/customer-login-api.md

```diff
@@ -19,21 +19,22 @@ You can use the Customer Login API in the following use cases:
 
 Storefront customers are logged in using the access point URL `/login/token/{token}`. The `{token}` must be a JSON Web Token (JWT) containing parameters for the customer login request signed by your application’s OAuth client secret. For more information on the OAuth protocol, see [OAuth](https://oauth.net/2/). 
 
-JWT is an industry-standard ([RFC 7519](https://tools.ietf.org/html/rfc7519)) for securely transmitting information between two parties. A JWT represents a sequence of base64url-encoded sections separated by dots (` . `).  The sections include the header, payload, and signature. For more details, see [Introduction to JSON Web Tokens](https://jwt.io/introduction/). 
+JWT is an industry standard ([RFC 7519](https://tools.ietf.org/html/rfc7519)) for securely transmitting information between two parties. A JWT is a sequence of base64url-encoded strings separated by dots (` . `).  The sections include the header, payload, and signature. For more details, see [Introduction to JSON Web Tokens](https://jwt.io/introduction/). 
 
-You are required to include the `channel_id` when using the login JWTs to embed checkout for headless storefronts. Default value = 1
+You are required to include the `channel_id` when using the login JWTs to embed checkout for headless storefronts. The default `channel_id` value is `1`.
 
-**Payload fields reference**
+### Customer Login JWT payload reference
 
 | Field Name | Type | Description |
-|-|-|-|
-| `iss` | string | Indicates the token's issuer. This is your application's Client ID.|
-| `iat` | integer| Time when the token was generated. This is a numeric value indicating the number of seconds since the [Unix epoch](http://en.wikipedia.org/wiki/Unix_time).|
+|:-----------|:-----|:------------|
+| `iss` | string | Indicates the token's issuer. This is your API account's client ID.|
+| `iat` | integer| The time when the token was generated. This is a numeric value indicating the number of seconds since the [Unix epoch](http://en.wikipedia.org/wiki/Unix_time).|
 | `jti` | string | A unique request ID (ex. uuid).|
 | `operation` | string | Must contain the string `"customer_login"`.|
-| `store_hash` | string | Store hash identifying the store you are logging into.|
-| `customer_id` | integer | ID of the customer you are logging in.|
-| `redirect_to` | string | Optional field containing a relative path for the shopper's destination after login. Will default to `/account.php`. |
+| `store_hash` | string | The store hash identifying the store the shopper is signing in to.|
+| `channel_id` | integer | Optional field containing the `channel_id` corresponding to the storefront or other sales channel the shopper is signing in to. |
```

Comment:
Other [channels](https://developer.bigcommerce.com/api-reference/b3A6MzU5MDQ0NDc-create-a-channel) may include Marketplace, POS, "marketing" (I'm not clear on the intended use her) or "other". I'm not sure how a customer would use the Customer Login API to auth against any non-storefront channels.

Consider removing "or other sales channel" from the description.

Reply:
@hatertron3000 @traci707 My thought is something like identifying a user who's a repeat customer purchasing on a social media channel. So they're an existing customer on the store's instagram channel. The implementation that's using the Customer Login API spins up in response to webhook events.  Hazy on details from there... but it seems like this is the kind of thing for which MSF / multichannel is trying to leave open the space of possibility.